great_expectations/execution_engine/sparkdf_execution_engine.py

```diff
@@ -169,12 +174,16 @@ def __init__(
         self._spark_config = spark_config
         self.spark = spark
 
+        azure_options: dict = kwargs.pop("azure_options", {})
+        self._azure_options = azure_options
```

Comment:
Very nitpicky but we could probably consolidate to one line? I'll defer to your judgement.

Reply:
@cdkini I did this on purpose (to follow what we do with `spark_config`).

Reply:
okay then please disregard! resolving